themes/README.rst

```diff
@@ -0,0 +1,146 @@
+#####################
+Comprehensive Theming
+#####################
+
+
+Comprehensive Theming lets you customize the appearance of your Open edX
+installation.  You can override Sass and CSS settings, images, or entire HTML
+templates.
+
+Eventually, Comprehensive Theming will obsolete existing theming mechanisms,
+but for now they co-exist peacefully. This document describes how to use 
+Comprehensive Theming, and also the changes you'll need to make to keep other
+theming mechanisms working.
+
+
+Creating a theme
+================
+
+A theme is a directory of assets.  You can create this directory wherever you
+like, it does not have to be inside the edx-platform directory.  The structure
+within this directory mirrors the assets in the edx-platform repo itself.
+Files you provide in your theme are used in place of the same-named files in
+edx-platform.  Here's a sample::
+
+    my-theme
+    └── lms
+        ├── static
+        │   ├── images
+        │   │   └── logo.png
+        │   └── sass
+        │       ├── _overrides.scss
+        │       ├── lms-main-rtl.scss
+        │       └── lms-main.scss
+        └── templates
+            ├── footer.html
+            └── header.html
+
+The top directory is named whatever you like.  This example uses "my-theme".
+The files provided here override the files in edx-platform.  In this case, the
+``my-theme/lms/static/sass/lms-main.scss`` file is used in place of the
+``edx-platform/lms/static/lms-main.scss`` file.
+
+
+Images
+------
+
+Images can be substituted simply by placing the new image at the right place
+in the theme directory.  In our example above, the lms/static/images/logo.png
+image is overridden.
+
+
+Sass/CSS
+--------
+
+Most CSS styling in Open edX is done with Sass files compiled to CSS.  You can
+override individual settings by creating a new Sass file that uses the existing
+file, and overrides the few settings you want.
+
+For example, to change the fonts used throughout the site, you can create an
+``lms/static/sass/_overrides.scss`` file with the change you want::
+
+    $sans-serif: 'Helvetica';
+
+The variables that can currently be overridden are defined in
+``lms/static/sass/base/_variables.scss``.
+
+**Note:** We are currently in the middle of a re-engineering of the Sass
+variables.  They will change in the future.  If you are interested, you can see
+the new development in the `edX Pattern Library`__.
+
+.. __: http://ux.edx.org/
+
+Then create ``lms/static/sass/lms-main.scss`` to use those overrides, and also
+the rest of the definitions from the original file::
+
+    // Our overrides for settings we want to change.
+    @import 'overrides';
+
+    // Import the original styles from edx-platform.
+    @import 'lms/static/sass/lms-main';
+
+Do this for each .scss file your site needs.
```

Comment:
> I like this idea very much, especially the parallelism between the sass lookup path and the template lookup path. @talbs this sounds good, right? Is one overrides file sufficient? With the current code, there can be separate overrides for separate sass files, but that's overkill, right?

Hmm, I'm not sure I'm following the use cases here. Mind chatting in person about what @mtyaka is proposing?